packages/near-membrane-base/src/environment.ts

```diff
@@ -1,30 +1,33 @@
 import {
     init,
+    CallableDefineProperty,
     CallableEvaluate,
     CallableInstallLazyDescriptors,
+    CallableSetPrototypeOf,
+    DistortionCallback,
+    GetSelectedTarget,
     GetTransferableValue,
     HooksCallback,
-    CallableSetPrototypeOf,
     Pointer,
-    CallableDefineProperty,
     ProxyTarget,
-    GetSelectedTarget,
+    SupportFlags,
 } from './membrane';
 
 const frameGlobalNamesRegExp = /^\d+$/;
-const ShouldTrapMutation = true;
-const ShouldNotTrapMutation = false;
 
-interface VirtualEnvironmentOptions {
+export interface VirtualEnvironmentOptions {
     // Blue connector factory
     blueConnector: typeof init;
+    // Optional distortion callback to tame functionalities observed through the membrane
+    distortionCallback?: DistortionCallback;
     // Red connector factory
     redConnector: typeof init;
-    // Optional distortion callback to tame functionalities observed through the membrane
-    distortionCallback?: (originalTarget: ProxyTarget) => ProxyTarget;
+    // Flag used for CSSStyleDeclaration objects in Safari 14
```

Comment:
This comment is too specific for this line now. I think it belongs here: https://github.com/salesforce/near-membrane/pull/190/files#diff-864f71e5394448a26d4291d300f74f3385a60601d574b7fd1a488efbeab85aedR127

Reply:
Cool